application/pr/viewer.jsx

```diff
@@ -46,6 +47,13 @@ export default class PRViewer extends React.Component {
             .then(() => this.setState({groupDetails, status: "ready"}))
             .catch(utils.catchHandler)
         ;
+        fetch(pp + "api/repos/" + [owner, shortName].join("/"), { credentials: "include"})
```

Comment:
looking at this in the overall context, shouldn't that promise be chained with the others? Otherwise, the page will assume it is "ready" even if that last request hasn't resolved yet

Reply:
addressed in https://github.com/w3c/ash-nazg/pull/163/commits/de3d7124249498eb751bdc52d3e463e88ac8bf55